docs/ebpf/concepts/rlimit.md

```diff
@@ -0,0 +1,88 @@
+# Resource Limits
+
+Creating eBPF objects (Maps, Programs, even BTF blobs) requires kernel memory
+allocation. Before kernel version 5.11, you could control the memory available
+to a process for creating eBPF objects by adjusting the process'
+`RLIMIT_MEMLOCK` rlimit value, visible through the `ulimit -l` command.
+
+!!! note ""
+    User space typically does *not* allocate or `mlock(2)` memory for the
```

Comment:
Again, interesting as a contributor, not so relevant as a user? Also the same link to lore.kernel.org in two subsequent paragraphs.